### PR TITLE
Adding the option to set PORT for RabbitMQ producer

### PR DESCRIPTION
**Pull request by microThread**

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/MaxwellConfig.java [hunk 1]
```diff
@@ -102,8 +102,8 @@
 
 	public String rabbitmqUser;
 	public String rabbitmqPass;
-	public int rabbitmqHost;
-	public String rabbitmqPort;
+	public String rabbitmqHost;
+	public int rabbitmqPort;
```
Comment (ashashwat) [inline src/main/java/com/zendesk/maxwell/MaxwellConfig.java]: This commit should have been squashed with the previous commit [8d1a31](https://github.com/zendesk/maxwell/pull/921/commits/8d1a31b20c5f26ef1f5fdd78cf662945f68e7403).

File: src/main/java/com/zendesk/maxwell/MaxwellConfig.java [hunk 2]
```diff
@@ -355,16 +357,17 @@ private void setup(OptionSet options, Properties properties) {
...
-		this.rabbitMqExchangeAutoDelete = fetchBooleanOption("rabbitmq_exchange_autodelete", options, properties, false);
-		this.rabbitmqRoutingKeyTemplate   = fetchOption("rabbitmq_routing_key_template", options, properties, "%db%.%table%");
-		this.rabbitmqMessagePersistent    = fetchBooleanOption("rabbitmq_message_persistent", options, properties, false);
+		this.rabbitmqVirtualHost    		= fetchOption("rabbitmq_virtual_host", options, properties, "/");
+		this.rabbitmqExchange       		= fetchOption("rabbitmq_exchange", options, properties, "maxwell");
+		this.rabbitmqExchangeType   		= fetchOption("rabbitmq_exchange_type", options, properties, "fanout");
+		this.rabbitMqExchangeDurable 		= fetchBooleanOption("rabbitmq_exchange_durable", options, properties, false);
+		this.rabbitMqExchangeAutoDelete 	= fetchBooleanOption("rabbitmq_exchange_autodelete", options, properties, false);
+		this.rabbitmqRoutingKeyTemplate   	= fetchOption("rabbitmq_routing_key_template", options, properties, "%db%.%table%");
+		this.rabbitmqMessagePersistent    	= fetchBooleanOption("rabbitmq_message_persistent", options, properties, false);
```
Comment (ashashwat) [inline src/main/java/com/zendesk/maxwell/MaxwellConfig.java]: Changing formatting of this whole block, should it really be part of this PR?

File: src/main/java/com/zendesk/maxwell/MaxwellConfig.java [hunk 3]
```diff
@@ -355,16 +357,17 @@ private void setup(OptionSet options, Properties properties) {
 		this.pubsubTopic 		 = fetchOption("pubsub_topic", options, properties, "maxwell");
 		this.ddlPubsubTopic  = fetchOption("ddl_pubsub_topic", options, properties, this.pubsubTopic);
 
-		this.rabbitmqHost           = fetchOption("rabbitmq_host", options, properties, "localhost");
-		this.rabbitmqUser			= fetchOption("rabbitmq_user", options, properties, "guest");
+		this.rabbitmqHost           		= fetchOption("rabbitmq_host", options, properties, "localhost");
+		this.rabbitmqPort 			= Integer.parseInt(fetchOption("rabbitmq_port", options, properties, "5672"));
```
Comment (ashashwat) [inline src/main/java/com/zendesk/maxwell/MaxwellConfig.java]: What will happen if I pass rabbitmq_port as say "abcd" i.e. invalid port or say "123456789" i.e. port, not in valid range?

**Comment by osheroff** [comment]
hokay 👍

**Comment by microThread** [comment]
Thank you and I'm sorry for extra code formatting. 🎉🍻

**Pull request merged**